tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -26,10 +26,17 @@ def _check_cpuid_x86(test_microvm, expected_cpu_count, expected_htt):
 
 
 def _check_cpu_features_arm(test_microvm):
-    expected_cpu_features = {
-        "Flags": "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
-        "asimdhp cpuid asimdrdm lrcpc dcpop asimddp ssbs",
-    }
+    if utils.get_instance_type() == "m6g.metal":
+        expected_cpu_features = {
+            "Flags": "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
+            "asimdhp cpuid asimdrdm lrcpc dcpop asimddp ssbs",
+        }
+    else:
```

Comment:
Maybe we can do sthg like:

```python
if m6g.metal:
   ...
if m7g.metal:
   ...
else:
   raise Exception("Unexpected instance")
```

that way it is explicit which flags belong to which instance type